### ISSUE TITLE
NTR: Use of intrauterine device (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7850]
New term request Use of intrauterine device (nadea3_210827111141): IUD  UMLS: 0344221

**Comment by pnrobinson** [comment]
@MickeySegal

Note I am coding this as distinct from  the recent Pregnancy despite IUD term, if this is not correct, please reopen this issue.

Adding new term: 
- ID: HP:6000167
- label: Use of intrauterine device
- parents: [Past obstetric history (HP:0032467)](https://hpo.jax.org/app/browse/term/HP:0032467)
- pmid: PMID:32491335
- def: Intrauterine devices (IUD) are commonly used for contraception. IUD complications are rare, but include perforation, pelvic infection, expulsion, missing strings, and difficult IUD removal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7850
- synonyms:
